### PR TITLE
Add Datasources to the Get Command

### PR DESCRIPTION
**Pull request by andyoneal**
this simply adds datasources as an option for the Get command, in addition to the existing views and workbooks.

### CODE REVIEW

**Comment by jacalata** [comment]
Code looks good. Can you add an e2e test in the e2e tests file and run it locally?

**Comment by andyoneal** [comment]
Done. Tests all passed.

**Review by jacalata — APPROVED** [review]

File: tests/e2e/language_tests.py [hunk 1]
```diff
@@ -74,17 +74,33 @@ def _publish_wb(self, file, name):
...
+        return _test_command(arguments)
+
     def _delete_wb(self, file):
         command = "delete"
         arguments = [command, "-w", file]
         _test_command(arguments)
 
+    def _delete_ds(self, file):
+        command = "delete"
+        arguments = [command, file, "--datasource"]
```
Comment (jacalata) [inline tests/e2e/language_tests.py:89]: does this option exist in delete?

File: tests/e2e/language_tests.py [hunk 2]
```diff
@@ -74,17 +74,33 @@ def _publish_wb(self, file, name):
         arguments = [command, file, "--name", name, "--overwrite"]
         return _test_command(arguments)
 
+    def _publish_ds(self, file, name):
```
Comment (jacalata) [inline tests/e2e/language_tests.py:77]: does this command work?

**Review by jacalata — CHANGES_REQUESTED** [review]
=========================== short test summary info 
FAILED tests/commands/test_geturl_utils.py::GeturlTests::test_get_workbook_name
E       AttributeError: type object 'GetUrl' has no attribute 'get_workbook_name'
tests/commands/test_geturl_utils.py:46: AttributeError

FAILED tests/commands/test_run_commands.py::RunCommandsTest::test_get_workbook
        mock_args.url = "/workbooks/workbook-name"
        mock_args.filename = "filename.twbx"
>       get_url_command.GetUrl.run_command(mock_args)

tests/commands/test_run_commands.py:122: 
tabcmd/commands/datasources_and_workbooks/get_url_command.py:66: in run_command
    Errors.exit_with_error(logger, message=_("tabcmd.get.extension.not_found"))

=================== 2 failed, 184 passed, 1 skipped in 1.00s

**Pull request merged**